tools/internal_ci/helper_scripts/requirements.linux_perf.txt

```diff
@@ -0,0 +1,5 @@
+tabulate>=0.8
+scipy>=1.0.0
+requests>=2.19.0
+PyJWT>=2.0.1
+cryptography>=3.3.1,<4.0.0
```

Comment:
nit: For CI dependencies, it's better to pin the package versions.

Reply:
Done